x/campaign/keeper/msg_create_campaign_test.go

```diff
@@ -126,6 +127,16 @@ func TestMsgCreateCampaign(t *testing.T) {
 			},
 			err: types.ErrFundCommunityPool,
 		},
+		{
+			name: "should prevent validation of msg with invalid metadata length",
```

Comment:
```suggestion
			name: "should fail when the change had too long metadata",
```
The message is already validated at this stage in the literal sense